lib/node-utils.js

```diff
@@ -588,15 +588,16 @@ function getTokenType(token) {
  * @returns {ESTreeToken}       the converted ESTreeToken
  */
 function convertToken(token, ast) {
-    const start = token.getStart(),
+    const start = (token.kind === SyntaxKind.JsxText) ? token.getFullStart() : token.getStart(),
+        end = token.getEnd(),
         value = ast.text.slice(start, token.end),
```

Comment:
I was reading through the code of typescript and noticed this method. I thought it looked a little cleaner since we use methods to access the starting point.

These are the methods to access token positions:
 - `getFullStart()` is the same as `.pos`
 - `getEnd()` is the same as `.end`
 - `getStart()` is the same as `.pos` but skipping over trivia and white space

If you want I can change the logic to use the properties to be more consistent with the rest of the code.